app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -43,6 +56,13 @@ def validate_import_vm
     Gem::Version.new(api_version) >= Gem::Version.new('4')
   end
 
+  def submit_configure_imported_vm_networks(userid, vm_id)
```

Comment:
I tend to agree with @matobet this logic should not normally be called on a VM, and the only reason it might be called is as part of this process.
Of course its more procedural than OOP but it seems to me it will be easier to understand this way.